plugins/generator-1.20.1/forge-1.20.1/templates/item/item.java.ftl

```diff
@@ -285,6 +290,7 @@ public class ${name}Item extends Item {
 
 	<#if data.enableRanged && data.shootConstantly>
 		@Override public void onUseTick(Level world, LivingEntity entity, ItemStack itemstack, int count) {
+			<#if data.rangedItemChargesPower>float pullingPower = 1;</#if>

```

Comment:
There is no need for this on shootConstantly.

What I meant is to disable the field in UI when this parameter is enabled.

We should avoid arbitrary code like this that is not needed in generators